### ISSUE TITLE
New install RPM, grafana does not respond

### ISSUE THREAD

**Issue by nozen** [grafana/grafana#5825]
* **I'm submitting a ...**
- [x ] Bug report
- [ ] Feature request
- [ ] Question / Support request: **Please do not** open a github issue. [Support Options](http://grafana.org/support/)

Please include this information:
- What Grafana version are you using? 3.1.1
- What datasource are you using? None
- What OS are you running grafana on? Redhat 7
- What did you do? Installed grafana from rpm
- What was the expected result? grafana should be accessible
- What happened instead? grafana does not open

Basically, installed grafana using the RPM. I started the grafana service and i can see it running. I followed the instructions on the grafana web site to do this.

When i browse the name or IP on port 3000 the website does not resolve. 

What am i missing or is this bug in this release?

I dont have the firewall running.

I'm using the default config ini.

**Comment by nozen** [comment]
Never mind, i noobed up the firewall.
